src/lib/modules/blockchain_connector/index.js

```diff
@@ -413,23 +413,12 @@ class BlockchainConnector {
       'get',
       '/embark-api/blockchain/transactions/:hash',
       (req, res) => {
-        const isRawTxHash = req.query.isRawTxHash && req.query.isRawTxHash === 'true';
-
-        if (isRawTxHash) {
-          self.getTransactionByRawTransactionHash(req.params.hash, (err, transaction) => {
-            if (err) {
-              self.logger.error(err);
-            }
-            res.send(transaction);
-          });
-        } else {
-          self.getTransaction(req.params.hash, (err, transaction) => {
-            if (err) {
-              self.logger.error(err);
-            }
-            res.send(transaction);
-          });
-        }
+        self.getTransactionByRawTransactionHash(req.params.hash, (err, transaction) => {
+          if (err) {
+            self.logger.error(err);
+          }
+          res.send(transaction);
+        });
```

Comment:
Removed the "fetch transaction" feature now and only decode raw transaction hashes as discussed in our call

Reply:
@jrainville notice that the "error handling" is still the same as before, I can still change this to actually sending a bad error code so it also arrives as an actual error in the client.

Let me know what you think.